src/chainparams.cpp

```diff
@@ -156,11 +156,11 @@ class CMainParams : public CChainParams {
         vSeeds.push_back(CDNSSeedData("bitcoinstats.com", "seed.bitcoinstats.com"));
         vSeeds.push_back(CDNSSeedData("xf2.org", "bitseed.xf2.org"));
 
-        base58Prefixes[PUBKEY_ADDRESS] = boost::assign::list_of(0);
-        base58Prefixes[SCRIPT_ADDRESS] = boost::assign::list_of(5);
-        base58Prefixes[SECRET_KEY] =     boost::assign::list_of(128);
-        base58Prefixes[EXT_PUBLIC_KEY] = boost::assign::list_of(0x04)(0x88)(0xB2)(0x1E);
-        base58Prefixes[EXT_SECRET_KEY] = boost::assign::list_of(0x04)(0x88)(0xAD)(0xE4);
+        base58Prefixes[PUBKEY_ADDRESS] = std::vector<unsigned char>(1,0);
+        base58Prefixes[SCRIPT_ADDRESS] = std::vector<unsigned char>(1,5);
+        base58Prefixes[SECRET_KEY] =     std::vector<unsigned char>(1,128);
+        base58Prefixes[EXT_PUBLIC_KEY] = boost::assign::list_of(0x04)(0x88)(0xB2)(0x1E).convert_to_container<std::vector<unsigned char> >();
```

Comment:
I'm terrified. This makes me yearn for the old days:
```
static const uint8_t ext_public_key[] = {0x04, 0x88, 0xB2, 0x1E};
base58Prefixes[EXT_PUBLIC_KEY] = std::vector<uint8_t>(ext_public_key, ext_public_key+sizeof(ext_public_key));
```
Boost doesn't add anything of value here.